docs/src/features.md

```diff
@@ -34,6 +34,8 @@ In addition to the basic "Goto Definition", the protocol also provides further r
 Additionally, the LSP's "Goto Definition" command can fall back to the built-in Sublime's "Goto Definition" if the `fallback` argument is set to `true`.
 This way, when there are no results found the built-in "Goto Definition" command will be triggered.
 
+To always open the results in a certain group, you can use the `group` argument.
```

Comment:
It doesn't appear to "always" open in specified group but rather it opens only in specified group if such group exists. So if there is only one group then basically `group` will be ignored.

At least that's how it seems to work from my testing.